### PR TITLE
Pfs 35 add write off

### PR DESCRIPTION
**Pull request by denialanderror**

### CODE REVIEW

File: cypress/e2e/adjust-invoice.cy.js [hunk 1]
```diff
@@ -28,7 +28,19 @@ describe("Adjust invoice form", () => {
         cy.get('.govuk-button').click();
 
         // navigation and success message
-        cy.url().should('include', "clients/3/invoices?success=CREDIT_MEMO");
+        cy.url().should('include', "clients/3/invoices?success=invoice-adjustment[CREDIT%20MEMO]");
```
Comment (kate-49) [inline cypress/e2e/adjust-invoice.cy.js:31]: Is there a reason you've gone for a space in this value? It looks a bit simpler when it was CREDIT_MEMO

File: finance-api/internal/service/ledger_entries.go [hunk 2]
```diff
@@ -12,53 +12,54 @@ import (
...
-		return err
-	}
+func (s *Service) validateAdjustmentAmount(adjustment *shared.CreateLedgerEntryRequest, balance store.GetInvoiceBalanceRow) error {
 	switch adjustment.AdjustmentType {
 	case shared.AdjustmentTypeAddCredit:
-		if int32(adjustment.Amount)-b.Outstanding > b.Initial {
-			return shared.BadRequest{Field: "Amount", Reason: fmt.Sprintf("Amount entered must be equal to or less than £%d", (b.Initial+b.Outstanding)/100)}
+		if int32(adjustment.Amount)-balance.Outstanding > balance.Initial {
+			return shared.BadRequest{Field: "Amount", Reason: fmt.Sprintf("Amount entered must be equal to or less than £%d", (balance.Initial+balance.Outstanding)/100)}
+		}
```
Comment (kate-49) [inline finance-api/internal/service/ledger_entries.go:47]: Is there any risk of us having issues with rounding as this is not money type?

**Review by kate-49 — APPROVED** [review]

File: cypress/e2e/adjust-invoice.cy.js (hunk 1 above)
Comment (denialanderror) [reply cypress/e2e/adjust-invoice.cy.js:31]: Ideally, we'd use an underscore everywhere but for some unknown reason, these values are stored in the database with spaces instead. We could have these with the underscore in but it means adding another translation function to the enum.

File: finance-api/internal/service/ledger_entries.go (hunk 2 above)
Comment (denialanderror) [reply finance-api/internal/service/ledger_entries.go:47]: It will round it to the nearest pound, which is all that's really needed in this instance.

**Pull request merged**